### DISCUSSION TITLE
Why no mariadb dependency in base package mingw-w64-qt6-static

### DISCUSSION THREAD

**Discussion by Selzig** [msys2/MINGW-packages#12191] [General]
Hello,

in the package details [mingw-w64-qt6-base-static 6.3.1-1](https://aur.archlinux.org/packages/mingw-w64-qt6-base-static) of the archlinux user repository,  we see a mingw-w64-mariadb-connector-c dependency which does not seem to be included in mingw-w64-qt6-static  whereas in mingw-w64-x86_64-qt6 we have the mingw-w64-x86_64-libmariadbclient optional dependency.  Do you know why ?

Best regards. Selzig.

**Comment by revelator** [comment]
i think its because it would add a dependency on a dynamic library, i dont think the mariadbconnector can be built statically ?.
